tests/version/test_markers.py

```diff
@@ -1497,7 +1497,7 @@ def test_only(marker: str, only: list[str], expected: str) -> None:
         ("", "~3.8", ""),
         ("<empty>", "~3.8", "<empty>"),
         ('sys_platform == "linux"', "~3.8", 'sys_platform == "linux"'),
-        ('python_version >= "3.8"', "~3.8", ""),
+        ('python_version == "3.8"', "~3.8", ""),
```

Comment:
I just changed this test because there is a duplicate of the original test some lines below.